src/core/ConfigManager.cpp

```diff
@@ -158,6 +158,16 @@ void ConfigManager::upgrade_1_1_91()
 }
 
 
+void ConfigManager::upgrade_1_3_0()
+{
+	// Adds the option of using the solo legacy behavior (muting Automation Tracks as well) if
+	// it isn't on the configuration file (safety check + upgrade)
+	if(value("app", "sololegacybehavior").isNull()) {
+		setValue("app", "sololegacybehavior", "0");
+	}
+}
```

Comment:
I don't think this upgrade function is necessary - we haven't written one when adding any other new values in the past. Instead we prefer to specify a default value, which covers more scenarios (e.g. when the configuration file doesn't yet exist). Upgrades are generally used instead to transform existing values.

Reply:
Sure, I can delete the `upgrade` method and just add a default value to every call to `ConfigManager::value` that uses this entry. I'm writing the changes atm and will push them!